app/imports/components/posts/NewPost.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+import { addPost } from '/imports/api/photosMethods.js';
+
+export class NewPost extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      photo: this.props.photo,
+      caption: ''
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({caption: event.target.value});
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    const photo = this.state.photo;
+    const caption = this.state.caption;
+
+    addPost.call({
+      photo,
+      caption,
+    });
+
+    return false;
+  }
+
+  render() {
+    return (
+      <section>
+        <img src={this.state.photo} />
+        <form onSubmit={this.handleSubmit}>
```

Comment:
actually remove the button and bind the onSubmit (as form can be submitted without any button from phone input view)